### PR TITLE
Added a cross button for deleting appointments by patient himself.

### PR DESCRIPTION
**Pull request by vedthakur5**

### CODE REVIEW

**Comment by harsh-not-haarsh** [comment]
Hey @vedthakur5 
Can you add some relevant screen shots?

**Comment by vedthakur5** [comment]
a
Screenshot 2021-03-17 161516
These are some screenshots

File: hc/urls.py [hunk 1]
```diff
@@ -14,14 +14,14 @@
     path('treatPatient/', treatPatientView.as_view(template_name="doctor/treat_patient.html"),
          name='prescription'),
     path('makeAppointment/', makeAppointment, name='makeAppointment'),
+    re_path(r'^cancelAppointment/(?P<pk>[0-9]+)/$', cancelAppointment, name='cancelAppointment'),
```
Comment (harsh-not-haarsh) [inline hc/urls.py:17]: if you can do this with path only

File: hc/views/views_patient.py [hunk 2]
```diff
@@ -152,3 +152,15 @@ def AddFamilyMemberView(request):
         form = AddFamilyMemberForm()
 
     return render(request, 'patient/add_member.html', {'form': form})
+
+
+def cancelAppointment(request, pk):
```
Comment (harsh-not-haarsh) [inline hc/views/views_patient.py:157]: rename the pk to apId or something like that

File: templates/main/index.html [hunk 3]
```diff
@@ -49,10 +49,14 @@ <h2 class="d-flex align-items-center">My Appointments</h2>
...
                         {% for appointment in appointments %}
-                        <li>Dr. {{appointment.doctor.user.first_name}} <span>{{appointment.time}}</span></li>
+                        <li>Dr.{{appointment.doctor.user.first_name}}<span>{{appointment.time}}</span><form action="{% url 'hc:cancelAppointment' appointment.id %}" method="post">
+                            {% csrf_token %}
+                            <button type="submit" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                        </form></li>
                         {% endfor %}
                     </ul>
                     {% endif %}
+                    <script src="{% static '/js/test.js' %}" type="text/javascript"></script>
```
Comment (harsh-not-haarsh) [inline templates/main/index.html:59]: remove this

**Review by harsh-not-haarsh — CHANGES_REQUESTED** [review]

**Comment by vedthakur5** [comment]
created a new pr #22

**Review by harsh-not-haarsh — APPROVED** [review]

**Pull request merged**